.ci/StarterRuleSet-AllRulesByCategory.groovy.txt

```diff
@@ -359,9 +359,6 @@ ruleset {
     NonFinalSubclassOfSensitiveInterface
     ObjectFinalize
     PublicFinalizeMethod
-    SystemExit {
-        doNotApplyToFileNames = 'pitest-survival-check-xml.groovy,error-prone-check.groovy'
-    }
```

Comment:
We suppress every groovy script, removal of this is better, we will most probably use groovy only for writing scripts.